manifests/moco-nodes.pp

```diff
@@ -21,14 +21,22 @@
 }
 
 # Linux on moonshot in mdc1 running taskcluster worker
-node /^t-linux64-(ms|xe)-\d{3}\.test\.releng\.(mdc1|mdc2)\.mozilla\.com$/ {
+node /^t-linux64-(ms|xe)-\d{3}\.test\.releng\.mdc1\.mozilla\.com$/ {
     $aspects          = [ 'low-security' ]
     $slave_trustlevel = 'try'
-    $taskcluster_worker_type  = 'gecko-t-linux-talos'
+    $taskcluster_worker_type  = 'gecko-t-linux-talos-tw'
     include fw::profiles::linux_taskcluster_worker
     include toplevel::worker::releng::taskcluster_worker::test::gpu
 }
 
+# Linux on moonshot in mdc2 running generic worker
```

Comment:
This comment should say "running taskcluster-worker, but will be migrated to generic-worker once bug X lands".

In other words, a reference to a bug (and relevant comment in bug) is useful here that shows that a migration is underway such that all taskcluster-worker workload is being moved from worker type `gecko-t-linux-talos` to `gecko-t-linux-talos-tw`, and that when that completes, `gecko-t-linux-talos` will then be used for generic-worker implementation of talos linux tasks.